cmd/update-main.go

```diff
@@ -62,15 +64,24 @@ EXAMPLES:
 `,
 }
 
-const releaseTagTimeLayout = "2006-01-02T15-04-05Z"
-
-const minioReleaseURL = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
+const (
+	minioReleaseTagTimeLayout = "2006-01-02T15-04-05Z"
+	minioReleaseURL           = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
+)
 
 func getCurrentReleaseTime(minioVersion, minioBinaryPath string) (releaseTime time.Time, err error) {
 	if releaseTime, err = time.Parse(time.RFC3339, minioVersion); err == nil {
 		return releaseTime, err
 	}
 
+	if !filepath.IsAbs(minioBinaryPath) {
+		// Make sure to look for the absolute path of the binary.
+		minioBinaryPath, err = exec.LookPath(minioBinaryPath)
+		if err != nil {
+			return releaseTime, err
+		}
+	}
+
 	// Looks like version is minio non-standard, we use minio binary's ModTime as release time.
 	fi, err := os.Stat(minioBinaryPath)
 	if err != nil {
```

Comment:
This never occurs.. it never occurs in an ideal world because exec.LookPath() would have failed already.